packages/extension/src/config.ui.ts

```diff
@@ -102,4 +103,7 @@ export const AdditionalSignInPrepend:
 
 export const AdditonalIntlMessages: IntlMessages = ADDITIONAL_INTL_MESSAGES;
 
-export const AmplitudeApiKey = AMPLITUDE_API_KEY;
+export const AmplitudeApiKey =
```

Comment:
@UrAvgDeveloper @jas2212 Please update this Dev and Prod Key
Once we got official ampiltude fetch account.